CHANGELOG.md

```diff
@@ -4,10 +4,22 @@
 
 ### New features
 
+### Bug fixes
```

Comment:
Yea, the thinking is that it will encourage people to put their changelog entries into one of our expected buckets when they make a contribution. This format is generally inspired by rubocop's changelog, and I guess I though they did that, but [maybe not actually](https://github.com/bbatsov/rubocop/commit/aeaab434ac32e33cc9e22839f2079e905590fce6)